src/gui/tray/ActivityListModel.cpp

```diff
@@ -45,6 +45,7 @@ QHash<int, QByteArray> ActivityListModel::roleNames() const
     QHash<int, QByteArray> roles;
     roles[DisplayPathRole] = "displaypath";
     roles[PathRole] = "path";
+    roles[AbsolutePathRole] = "abspath";
```

Comment:
I think I'd prefer "absolutePath" (and too bad displaypath isn't camel cased...).

Reply:
✔️